### PR TITLE
Add path for Neovim on Windows

### PR DESCRIPTION
**Pull request by rofrol**
Describe the details of your PR ...

### CODE REVIEW

**Comment by stappersg** [comment]
Happy Newyear

I do wish that this pull request gets this more attention than is comment. Like `merged`   :-)

File: README.md [hunk 1]
```diff
@@ -103,6 +103,7 @@ Add a vim-plug section to your `~/.vimrc` (or `~/.config/nvim/init.vim` for Neov
 ```vim
 " Specify a directory for plugins
 " - For Neovim: ~/.local/share/nvim/plugged
+" - For Neovim on Windows: ~/AppData/Local/nvim/autoload
```
Comment (janlazo) [inline README.md]:
```md
- For Neovim: expand(stdpath('config') . '/nvim/plugged')
```

Comment (junegunn) [reply README.md]: `stdpath` is new to me but it looks nice. `stdpath('config')` on my system returns `/Users/jg/.config/nvim`, while `stdpath('data')` returns `/Users/jg/.local/share/nvim`. Should we use `stdpath(data)` without `expand` instead?

Comment (janlazo) [reply README.md]: `expand` is only for the backslash pathsep on Windows because of `noshellslash` but it should be redundant for vim-plug.

Comment (janlazo) [reply README.md]: >  Should we use `stdpath(data)` without `expand` instead?

Sure. `stdpath('data') . '/plugged'`.

Comment (rofrol) [reply README.md]: `stdpath('data') . '/plugged'` works. But `init.vim` is still loaded from `~/AppData/Local/nvim/init.vim`.

**Comment by rofrol** [comment]
Also neovim dev on Windows still creates `~/AppData/Local/nvim-data` directory.

**Comment by janlazo** [comment]
```diff
diff --git a/README.md b/README.md
index 2bda1a5..1fa2b8b 100644
--- a/README.md
+++ b/README.md
@@ -90,7 +90,7 @@ $uri = 'https://raw.githubusercontent.com/junegunn/vim-plug/master/plug.vim'
 
 ### Usage
 
-Add a vim-plug section to your `~/.vimrc` (or `~/.config/nvim/init.vim` for Neovim):
+Add a vim-plug section to your `~/.vimrc` (or `stdpath('config') . '/init.vim'` for Neovim):
 
 1. Begin the section with `call plug#begin()`
 1. List the plugins with `Plug` commands
```

**Comment by rofrol** [comment]
@janlazo 

```diff
> -Add a vim-plug section to your `~/.vimrc` (or `~/.config/nvim/init.vim` for Neovim):
> +Add a vim-plug section to your `~/.vimrc` (or `stdpath('config') . '/init.vim'` for Neovim):
```

Why this change? Yesterday I have downloaded dev version of nvim and it still reads config from `~/AppData/Local/nvim/init.vim`.

**Pull request merged**